byoqm/main.py

```diff
@@ -3,6 +3,7 @@
 from byoqm.runner import Runner
 from byoqm.visuals.dashboard import Dashboard
 from visuals import line
+import click
 
 # Assumes the project is being run from the root of the repository
 MODELS_PATH = "models"
```

Comment:
I think deleting this was missed when we introduced `runner.py` as this variable did the same as `_MODEL_DIR` does now